eng/common/native/CommonLibrary.psm1

```diff
@@ -230,7 +230,7 @@ function New-ScriptShim {
     # WinShimmer is a small .NET Framework program that creates .exe shims to bootstrapped programs
     # Many of the checks for installed programs expect a .exe extension for Windows tools, rather
     # than a .bat or .cmd file.
-    # Source: https://github.com/dotnet/arcade/tree/master/src/WinShimmer
+    # Source: https://github.com/dotnet/arcade/tree/main/src/WinShimmer
     if (-Not (Test-Path "$ShimDirectory\WinShimmer\winshimmer.exe")) {
       $InstallStatus = DownloadAndExtract -Uri "$BaseUri/windows/winshimmer/WinShimmer.zip" `
                                           -InstallDirectory $ShimDirectory\WinShimmer `
```

Comment:
changes to `eng/common` folder will be overwritten. If you'd like to contribute and make this change, please change it here: https://github.com/dotnet/arcade/blob/main/eng/common/native/CommonLibrary.psm1

Reply:
Thank you @vlada-shubina &mdash; I reverted these changes in [commit `d62c154447`](https://github.com/dotnet/msbuild/pull/8437/commits/d62c154447cf10d8c2b09d9b7e3f02c5c9643ee2), and have pushed that along with the change you suggested below (in [commit `00bdb0046c`](https://github.com/dotnet/msbuild/pull/8437/commits/00bdb0046c95dfacd8722ab3f306260feeefc20e)).